api/src/pcapi/core/mails/transactional/sendinblue_template_ids.py

```diff
@@ -45,6 +45,9 @@ class TransactionalEmail(Enum):
     EMAIL_DUPLICATE_BENEFICIARY_PRE_SUBCRIPTION_REJECTED = Template(
         id_prod=80, id_not_prod=19, tags=["jeunes_compte_refuse_doublon"], use_priority_queue=True
     )
+    BOOKING_SOON_TO_BE_EXPIRED_TO_BENEFICIARY = Template(
```

Comment:
(détail / nommage) 

```suggestion
    BOOKINGS_SOON_TO_BE_EXPIRED_TO_BENEFICIARY = Template(
```

... car il y a plusieurs réservations.

Reply:
pour des raisons d'harmonie de nommage les template booking sont au singulier

Reply:
Pourtant, le template suivant contient un "s" : EXPIRED_BOOKING**S**_TO_BENEFICIARY